reactive/vpe_router.py

```diff
@@ -35,8 +35,10 @@ def validate_config():
     except Exception as e:
         remove_state('vpe.configured')
         set_state('blocked', 'validation failed: %s' % e)
-    else:
```

Comment:
It's a good point, moving the set_state inside of the try block is probably best